src/app/shared/components/profile-status-bar.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component } from '@angular/core';
```

Comment:
we need component folder -> 4 files in it as we have discussed

Reply:
If possible, I would prefer not to divide small components into files (up to 100-150 lines), for me this looks like pollution and overengineering.